mlflow/fastai.py

```diff
@@ -405,7 +405,7 @@ def _run_and_log_function(self, original, args, kwargs, unlogged_params, callbac
             callbacks += list(args[callback_arg_index])
             tmp_list[callback_arg_index] += [__MLflowFastaiCallback(self)]
             args = tuple(tmp_list)
-        elif "callbacks" in kwargs:
+        elif kwargs.get("callbacks"):
```

Comment:
Fixing this logical error since I caught it in https://github.com/mlflow/mlflow/pull/3735/files#r530146856 as well